apps-client/mobile/components/profile/profile-select.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import { Pressable, View } from 'react-native'
+
+import { Avatar, AvatarImage } from '@pple-today/ui/avatar'
+import { Icon } from '@pple-today/ui/icon'
+import { ImagePickerSuccessResult, launchImageLibraryAsync } from 'expo-image-picker'
+import { Pencil } from 'lucide-react-native'
+
+import { usePhotoLibraryPermission } from '@app/libs/use-permission'
+
+import { AvatarPPLEFallback } from '../avatar-pple-fallback'
+
+interface ProfileSelectProps {
+  originalImage?: string
+  imagePickerResult?: ImagePickerSuccessResult
+  onChangeImagePickerResult: (imagePickerSuccessResult: ImagePickerSuccessResult) => void
```

Comment:
This should be `ImagePickerAsset` instead of `ImagePickerSuccessResult`
```suggestion
  value?: ImagePickerAsset
  onChange: (asset: ImagePickerAsset) => void
```

Since the type of result is only canceled and we already handle that in the component
```ts
export type ImagePickerSuccessResult = {
  canceled: boolean
  assets: ImagePickerAsset[] | null;
};
```